tests/test_DataFrameWrapper.py

```diff
@@ -141,3 +142,31 @@ def test_to_zarr_with_metadata(
     )
 
     assert hl_dataset.attrs == expected_hl_dataset_with_metadata.attrs
+
+
+@pytest.mark.parametrize(
+    argnames="out_path",
+    argvalues=(
+        str(""),
+        str("hl.zarr"),
+    ),
+    ids=(
+        "out_directory",
+        "out_zarr",
```

Comment:
The `ids` are just for us, so they can be more human-friendly, how about:

```suggestion
        "directory",
        ".zarr file",
```